app/components/PageNotFound/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './style.scss';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const PageNotFound = () => {
+  return (
+    <div className="col-8 col-centre text-center">
```

Comment:
No col-center required. use offset classes. Let it be col-12 and col-md-8. Also move the col classes to the custom404 component.